tdrs-backend/tdpservice/reports/serializers.py

```diff
@@ -44,3 +44,20 @@ def create(self, validated_data):
     def update(self, instance, validated_data):
         """Throw an error if a user tries to update a report."""
         raise ImmutabilityError(instance, validated_data)
+
+
+class DownloadReportFileSerializer(serializers.ModelSerializer):
```

Comment:
If you take my suggestions for the detail endpoint then this is no longer needed.